content/en/project/deepelectrode-mapper/index.md

```diff
@@ -0,0 +1,96 @@
+---
+type: "project" # DON'T TOUCH THIS ! :)
+date: "2025-05-20" # Date you first upload your project.
+# Deep Electrode Mapper: Using Deep Learning to Localize EEG Electrodes' Coordinates
+
+# List the names of the collaborators within the [ ]. If alone, simple put your name within []
+names: [Joel P. Diaz-Fong, Lucas Vidal Murakami, Aijia Ivy Zhong]
+
+# Your project GitHub repository URL
+github_repo: https://github.com/JOEwithanL/DeepElectrodeMapper
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/PSY6983-2021/project_template), click `manage topics`.
+# Please only lowercase letters
+tags: [electrode localization, deep learning, clustering, preprocessing tools]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects.
+
+summary: "Accurate source localization in EEG requires electrode's coordinates. However, currently this process requires skill and experience in dealing with different methods and softwares, which can be expensive, laborious and time consuming (Tveter et al., 2024). The Deep Electrode Mapper’s objective is to solve the electrodes’ localization issue by implementing deep learning to segment the electrodes from multiple image formats and find their coordinates via clustering. Currently the project is still in progress, and its development will be shared via its repository. 
+
+References:
+
+Tveter, M., Tveitstøl, T., Nygaard, T., Kulashekhar, S., Bruña, R., Hammer, H. L., Hatlestad-Hall, C., & Haraldsen, I. R. H. (2024). EEG electrodes and where to find them: automated localization from 3D scans. Journal of Neural Engineering, 21(5), 056022."
+
+# If you want to add a cover image (listpage and image in the right), add it to your directory and indicate the name
+# below with the extension.
+image: ""
```

Comment:
please add a cover image for your project. The file (png or jpg, ideally small <1MB) should live alongised the `index.md` file.